### ISSUE TITLE
The payload type is not correct when the module sends notifications

### ISSUE THREAD

**Issue by xuepgao** [JanLoebel/mmm-touchgestures#2]
Hi @JanLoebel 

Thanks your quick response. 

The payload type is not correct when the module sends notifications. I hope I can correctly specify the payload type, e.g., integer, string, or a certain type.

This issue is not a urgent case. It's up to your time for helping me check logs.

my mmm-touchgestures config is as following:
[https://drive.google.com/open?id=1bnHE5jM73PZvyhA1NE7pvqWVE7x4VjhK](https://drive.google.com/open?id=1bnHE5jM73PZvyhA1NE7pvqWVE7x4VjhK)

MagicMirror startup and mmm-touchgestures running log are as followings:
[https://drive.google.com/open?id=11hvUqeDiEQ37z-8-OBWJhwgSV6auRaQl](https://drive.google.com/open?id=11hvUqeDiEQ37z-8-OBWJhwgSV6auRaQl)

MagicMirror web page dev tool log is as following:
[https://drive.google.com/open?id=10eBY4Xi7_UiO4THsnFdP2IhpiaK4EQTk](https://drive.google.com/open?id=10eBY4Xi7_UiO4THsnFdP2IhpiaK4EQTk)

Thanks again!

**Comment by JanLoebel** [comment]
Hey, I had a first glimpse and could not reproduce the error, very wired. I will provide a small example with a docker compose file, etc.. soon. But I think we have to further investigate why it don't work on your machine.

**Comment by xuepgao** [comment]
@JanLoebel 

Thank you for troubleshooting about this issue.

To simplify the complexity of the issue, I removed/reinstalled mmm-touchgestures and deleted other modules in config.js. Now there are 7 modules config (clock, calendar_monthly, currentweather, MMM-logging, mmm-touchgestures,MMM-pages, MMM-page-indicator) in config.js:
```
		{
			module: 'MMM-pages',
			name: "MMM-pages",
			classes: 'default',
			position: "bottom_bar",
			config: {
				animationTime: 0,
				modules:
					[
						["clock"],
						["calendar_monthly"],
						["currentweather"],
					],
				fixed: 
				["mmm-touchgestures", "MMM-page-indicator"]
			}
		},

		{
			module: 'MMM-page-indicator',
			name: 'MMM-page-indicator',
			classes: 'default',
			position: 'bottom_bar',
			config: {
				activeBright: true,
				inactiveHollow: true,
				pages: 3,
			}
		}
```
But the problem still exists.  You can find running log from the following link:
[https://drive.google.com/open?id=1GyZ9nBCuVtkEcVD3ICjj5_IAfNvnxZMi](https://drive.google.com/open?id=1GyZ9nBCuVtkEcVD3ICjj5_IAfNvnxZMi)

If you need me to cooperate on this , I can share my desktop to you using Webex. If you are willing to do that, let me know your convenient local time.

Have a nice day!

**Comment by xuepgao** [comment]
Hi @JanLoebel 

I installed another modules MMM-TouchNotifications and the same issue appeared. So I'm sure the root cause is in the MMM-pages.

Then I found the bug fixed log:
[https://github.com/edward-shen/MMM-pages/commit/67d142452d67d44426952017f35ff4252c901b92](https://github.com/edward-shen/MMM-pages/commit/67d142452d67d44426952017f35ff4252c901b92)

8 days ago..., Oh, it's my fault.

Thank you again! 
I'll close the issue.
